src/containers/BooksList/BooksList.js

```diff
@@ -1,15 +1,24 @@
-import { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
 import Book from '../../components/book/index';
+import { REMOVE_BOOK } from '../../actions';
+import store from '../../reducers/index';
```

Comment:
- [ ] I see you are importing the store object to the components that uses dispatch. 
This is also a bad practice. if you wanna dispatch an action to the reducer, you should use the useDispatch hook or use connect with a mapDispatchToProps fucntion.

Reply:
Resolved